app/views/providers/delegated_confirmation/index.html.erb

```diff
@@ -1,14 +1,9 @@
 <% content_for(:panel) do %>
-  <div class="govuk-panel govuk-panel--confirmation">
-    <h1 class="govuk-panel__title">
-      <%= t ".page-heading" %>
-    </h1>
-    <div class="govuk-panel__body">
-    <%= t ".sub_title" %>
-    <br>
+  <%= govuk_panel(
+    title_text: t(".page-heading")) do %>
+    <%= t ".sub_title" %><br>
```

Comment:
A very minor thing, and it's not your code because it was already in here, but I think having `<%= t(".sub_title") %>` is a little more readable than without brackets and it seems to be what we're using most of the time elsewhere in the code. Might be worth changing just for consistency

Reply:
Thank you. I have fixed this now.